flow/compositor_context.h

```diff
@@ -24,12 +24,16 @@ class LayerTree;
 enum class RasterStatus {
   // Frame has successfully rasterized.
   kSuccess,
-  // Frame needs to be resubmitted for rasterization. This is
-  // currently only called when thread configuration change occurs.
+  // Frame is submitted twice. This is currently only used when
```

Comment:
What does frame with the same layer tree mean and how is it different from the same frame being submitted twice? It might be worth clarifying the difference here.

Reply:
Done.